myhdl/_compat.py

```diff
@@ -31,3 +34,7 @@ def to_str(b):
 
     to_bytes = _identity
     to_str = _identity
+
+def ast_parse(s):
+    return compile(s, '<string>', 'exec', \
+                   print_function.compiler_flag|division.compiler_flag|PyCF_ONLY_AST)
```

Comment:
I think this approach is wrong. This function will always parse the AST with print_function and division enabled even if the particular source file did not import print_function/division from `__future__`